docs/TLS.md

```diff
@@ -4,13 +4,66 @@ The Percona Operator for MySQL uses Transport Layer
 Security (TLS) cryptographic protocol for the following types of communication:
 
 * Internal - communication between Percona Server for MySQL instances,
-* External - communication between the client application and ProxySQL.
+* External - communication between the client application and the cluster.
 
 The internal certificate is also used as an authorization method.
 
-TLS security can be configured in several ways. By default, the Operator
-generates long-term certificates automatically if there are no certificate
-secrets available. But certificates can be generated manually as well.
+TLS security can be configured in several ways.
+
+* By default, the Operator *generates long-term certificates* automatically if
+    there are no certificate secrets available.
+   
+    ??? note "The Operator's self-signed issuer is local to the Operator Namespace"
+        This self-signed issuer is created because Percona Distribution for MySQL
+        requires all certificates issued by the same source.
+
+* The Operator can use a specifically installed *cert-manager*, which will
```

Comment:
I think currently even cert manager issues certificates are for 10 years.
@cap1984 can you comment ^

Reply:
@tplavcic sorry, `duration: 8760h0m0s` that is one year.

Reply:
nice, thanks for checking!